src/utilities/theme/tokens.ts

```diff
@@ -11,6 +11,7 @@ export const Tokens = {
   modalShadow:
     '0px 26px 80px var(--p-shadow-from-dim-light), 0px 0px 1px var(--p-shadow-from-dim-light)',
   topBarShadow: '0 2px 2px -1px var(--p-shadow-from-direct-light)',
+  buttonShadow: '0px 1px 0px rgba(0, 0, 0, 0.05)',
```

Comment:
I wonder if this should just be using the `border` css variable? Understanding this is probably not what is defined in Figma.